packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -288,8 +292,15 @@ export function FlatTermSelector( { slug, __nextHasNoMarginBottom } ) {
 		singularName
 	);
 
+	const Wrapper = ( { children } ) =>
```

Comment:
I think this is creating a new component every time it's re-rendered. It's causing the focus to be lost after adding a new tag. Since this is a regression, let's fix this before RC 3 🙏.